e2e-polybft/txpool_test.go

```diff
@@ -159,6 +161,36 @@ func TestE2E_TxPool_Transfer_Linear(t *testing.T) {
+
+	customArtifact, err := artifact.DecodeArtifact([]byte(smJSON))
+	require.NoError(t, err)
+
+	sidechainKey, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	cluster := framework.NewTestCluster(t, 4,
+		framework.WithPremine(types.Address(sidechainKey.Address())),
```

Comment:
Does it make sense to shorten the epoch? Although this is not related to waiting epoch ending blocks. Anyhow if you feel it would be beneficial to shorten epoch, you can use `framework.WithEpochSize()` function.

Reply:
The size of epoch does not really matter for the test